diff --git a/web/test/memba_web/router_test.exs b/web/test/memba_web/router_test.exs
index 2590763..e3f86bd 100644
--- a/web/test/memba_web/router_test.exs
+++ b/web/test/memba_web/router_test.exs
@@ -141,8 +141,7 @@ defmodule MembaWeb.RouterTest do
       assert %{
                path_params: %{},
                pipe_through: [:browser, :club_member_required],
-               phoenix_live_view:
-                 {MembaWeb.MemberInvitationLive.New, :new, _opts, _live_session},
+               phoenix_live_view: {MembaWeb.MemberInvitationLive.New, :new, _opts, _live_session},
                plug: Phoenix.LiveView.Plug,
                plug_opts: :new,
                route: "/members/invitations/new"
