diff --git a/web/test/memba_web/router_test.exs b/web/test/memba_web/router_test.exs
index a09e19b..25697cf 100644
--- a/web/test/memba_web/router_test.exs
+++ b/web/test/memba_web/router_test.exs
@@ -139,7 +139,8 @@ defmodule MembaWeb.RouterTest do
       assert %{
                path_params: %{"message_id" => "message-123"},
                pipe_through: [:browser, :club_member_required],
-               phoenix_live_view: {MembaWeb.MemberMessageDeliveryLive.Show, nil, _opts, _live_session},
+               phoenix_live_view:
+                 {MembaWeb.MemberMessageDeliveryLive.Show, nil, _opts, _live_session},
                plug: Phoenix.LiveView.Plug,
                plug_opts: nil,
                route: "/messages/:message_id/delivery"
