diff --git a/web/priv/repo/migrations/20260713073147_add_verification_state_to_membership_person_email_addresses.exs b/web/priv/repo/migrations/20260713073147_add_verification_state_to_membership_person_email_addresses.exs
new file mode 100644
index 0000000..72c003a
--- /dev/null
+++ b/web/priv/repo/migrations/20260713073147_add_verification_state_to_membership_person_email_addresses.exs
@@ -0,0 +1,7 @@
+defmodule Memba.Repo.Migrations.AddVerificationStateToMembershipPersonEmailAddresses do
+  use Ecto.Migration
+
+  def change do
+
+  end
+end
